src/components/sample/sample.test.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { expect } from 'chai';
-import { shallow } from 'enzyme';
+import { render } from 'enzyme';
 import Sample from './sample';
 
 describe('Sample', () => {
   it('should have the content', () => {
-    const sample = shallow(<Sample />);
+    const sample = render(<Sample />);
```

Comment:
Shallow rendering would do here.. if it's to contain you are checking for here. it's faster and doesn't render the full dom.